storage/overlays/eks/storage-class.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: onepanel
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: io1
```

Comment:
I think regular `gp2` SSD volume would suffice here and we can remove `iopsPerGB`